layer.py

```diff
@@ -88,11 +88,19 @@ def __init__(self):
 
 
 class LayerSource(object):
+    EXTERNAL_RESOURCE_EXPRESSIONS = [
```

Comment:
See https://github.com/opengisch/QField/pull/3851/files#diff-ad0acdd865d66f2f32f6ac8d65b8ce09d91d5b45e79416e408bd8809896c5a2fR166 on how the # ~ tags ~ # are used.

Reply:
Let's wait on this before merge.

Reply:
@suricactus , ya